config/initializers/figaro.rb

```diff
@@ -21,7 +21,8 @@
   'GOV_DELIVERY_SERVER',
   'ES_URL',
   'ES_CLIENT_CERT_PATH',
-  'ES_CLIENT_KEY_PATH'
+  'ES_CLIENT_KEY_PATH',
+  'GIDS_HOST'
```

Comment:
Also whatever key is chosen, should also go in `config/application.yml.example`, and in `Jenkinsfile`